### ISSUE TITLE
GemPy install seems to disable anaconda envirconment

### ISSUE THREAD

**Issue by cmakoaz** [cgre-aachen/gempy#717]
I have been trying to create a working anaconda environment for GemPy and keep running into a problem once I get to installing GemPy. After creating a new environment in Anaconda Navigator, I use the following in conda prompt:

```
conda install theano
conda install -c conda-forge gempy
```

theano seems to install fine but after gempy is installed I get a long error output that I've attached to this post. The error seems to be related to not having Microsoft Visual Studio on my machine. Is that really the case?

Could this be due to installing theano and gempy with conda rather than pip? I have found that mixing pip and conda (as is suggested in the install documentation) disables the coding environment I like to use (Spyder), so I have tried to be consistent in using conda. 

Hopefully this question is not too basic. Help is appreciated!

[gempy_error.txt](https://github.com/cgre-aachen/gempy/files/9448015/gempy_error.txt)

**Comment by Japhiolite** [comment]
Hi @cmakoaz 

no worries, that's not a too basic question. Depending on how uptodate the conda feedstock of gempy is, it should not be a problem.
An alternative is to install theano with conda (necessary as this will install all the relevant dependencies for getting theano to work), and then install gempy with pip in the same environment.

However, it looks like you need SDK...which I don't know why it is necessary for gempy. So you could either [try to install SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/) and rerun your installation via conda, or try first installing theano via conda and then gempy via pip.

**Comment by cmakoaz** [comment]
I did install SDK, but the same errors are persisting. The error output recognizes a version of SDK is there, but there is a lot more of the error that has to do with Microsoft Visual Studio (same as the error message in the original post). I have tried installing Visual Studio (2022 and 2017 versions) but I am a little lost on what I should try. Is there a dependency of gempy or theano on Microsoft Visual Studio, or something else going on? Thanks!

**Comment by Japhiolite** [comment]
There's no gempy dependency on VS. The conda environment should also not be affected by that Message.
Depending on the environment, I sometimes also get SDK and VS warnings and errors. While bothersome, they are not corrupting the environment.

Another option, try installing theano via conda as you did and gempy via pip.

**Comment by AlexanderJuestel** [comment]
@cmakoaz do you still have issues installing GemPy?

**Comment by cmakoaz** [comment]
I was able to get GemPy running successfully using conda, and it is working with the Spyder coding environment. I was a while ago now, but I believe I installed theano via conda and gempy via pip as you said. When I activate the environment it spits out a bunch of errors, but (also like you said) they do not seem to be affecting the functioning of gempy. I was able to work through several of the tutorials and modify the code, etc.
Thanks!
